private/browse.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+    /*
+    *
+    * Yabe Comma-separated values database interface library.
```

Comment:
You didn't change this line :) This is not a CSV interface library. It's a database browsing interface library